src/python/WMCore/MicroService/MSTransferor/MSTransferor.py

```diff
@@ -72,6 +74,11 @@ def __init__(self, msConfig, logger=None):
         """
         super(MSTransferor, self).__init__(msConfig, logger=logger)
 
+        # persistent area for site list processing
+        wdir = '{}/storage'.format(os.getcwd())
+        self.storage = self.msConfig.get('persistentArea', wdir)
```

Comment:
We need to ensure that this area is persistent across POD restarts, so we do not lose data accidentally.
I remember we used to use something like `/data/srv/state/` for database related data.

Reply:
In addition to my previous comment, which I believe to be still pending, please make this directory configurable in the microservice configuration file.

Reply:
Please also provide the relevant services_config relevant MR for this new parameter.

Reply:
I would suggest adding a log record on this as well, e.g.: "Using directory /blah as workflow persistent area."